### DISCUSSION TITLE
Why are my GitHub Actions constantly hitting secondary rate limits?

### DISCUSSION THREAD

**Discussion by frencojobs** [community/community#28452] [Actions]
Yesterday, I ran into 3 secondary rate limits from different repos, which makes the total to be 4 combining with the one I had in earlier days. I can accept the limit if I'm calling the API too frequently or even abusing the API in malicious ways. But I'm barely running GitHub Actions in like days. And it doesn't happen when running Actions, only when my Action tries to push code that it happens.

I **really** don't think it's possible for me to go over the limits. I'm not sure where to look for my usage in detail but here's the billing page in settings, and it shows 0 for everything. Plus my repos are public so.

Here is one of the runs that failed, https://github.com/frencojobs/fenceparser/runs/7666535929?check_suite_focus=true. I use https://github.com/changesets/action to run `git push` command from inside the Action.

**Comment by byrneh** [comment]
@frencojobs there are dfferent types of rate limits, see [#rate-limiting](https://docs.github.com/en/rest/overview/resources-in-the-rest-api#rate-limiting). 
Additionally there is some general guidance around secondary rate limits here [#dealing-with-secondary-rate-limits](https://docs.github.com/en/rest/guides/best-practices-for-integrators#dealing-with-secondary-rate-limits).  The scondary rate limit criteria is not based solely on volume of request, but on the complexisty and type of requests.

**Reply by johnbent** [reply]
This is super-annoying. I run a volunteer non-profit webpage for the preservation of an endangered language. We have a volunteer group who consults about words which have not yet been documented. We used to use debugle for these discussions but debugle closed. No problem; github is awesome and free. But I wrote a script to automatically create issues in github from the debugle issues and I keep hitting the secondary rate limit. It shouldn't be a problem because I only need to create about 100 issues with 5 comments each. But I keep hitting this limit. It wouldn't be a problem if I could query the limit and then sleep accordingly. But when I try to query, all my rates look OK!!! Argh. :(

~/tekinged.com/scripts/debugle> curl -H "Accept: application/vnd.github+json" -H "Authorization: Bearer $GH_OATH" https://api.github.com/rate_limit | grep remaining
"remaining": 4715,
"remaining": 30,
"remaining": 5000,
"remaining": 5000,
"remaining": 100,
"remaining": 1000,
"remaining": 10000,
"remaining": 15000,
"remaining": 100,
"remaining": 4715,
